tests/test_gen_addon_readme.py

```diff
@@ -162,7 +162,10 @@ def test_rst_error(tmp_path):
                 cmd + opts, stderr=subprocess.STDOUT, universal_newlines=True
             )
         except subprocess.CalledProcessError as e:
-            assert "Title level inconsistent" in e.output
+            assert (
```

Comment:
I am not happy with this change, but different errors were raised for each python version 😢